gradle/libs.versions.toml

```diff
@@ -74,7 +77,9 @@ org-robolectric-robolectric = { group = "org.robolectric", name = "robolectric",
 espresso-core = { group = "androidx.test.espresso", name = "espresso-core", version.ref = "espresso-core" }
 io-mockk = { group = "io.mockk", name = "mockk", version.ref = "io-mockk" }
 org-jetbrains-kotlinx = { group = "org.jetbrains.kotlinx", name = "kotlinx-coroutines-android", version.ref = "org-jetbrains-kotlinx" }
-com-pingidentity-signals = { group = "com.pingidentity.signals", name = "android-sdk", version.ref = "com-pingidentity-signals" }
+com-pingidentity-signals  = { group = "com.pingidentity.signals", name = "android-sdk", version.ref = "com-pingidentity-signals" }
+appcompat = { group = "androidx.appcompat", name = "appcompat", version.ref = "appcompat" }
```

Comment:
can you check this both needed or not...

Reply:
Yes, looks like both `appcompat` and `material` are needed